src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  hasError: boolean;
+  closeNotification: () => void;
+}
+
+export const ErrorNotification: React.FC<Props> = React.memo(({
+  hasError,
+  closeNotification,
+}) => (
+  <div
+    data-cy="ErrorNotification"
+    className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: !hasError },
+    )}
+  >
+    <button
+      data-cy="HideErrorButton"
+      type="button"
+      aria-label="Close notification"
+      className="delete"
+      onClick={closeNotification}
+    />
+
+    Unable to add a todo
+    <br />
+    Unable to delete a todo
+    <br />
+    Unable to update a todo
```

Comment:
It's not clear why I see all this errors if I just got an error while todos loading (and I don't see message about loading)

Reply:
@OleksiiBlezniuk I implement errors message in second task, because it is not clear to me how to imitate such errors,  in this task I have not such methods as add, delete, update. So this one it is only like a dummy

Reply:
But I will transfer this one from second, so it's okay